### ISSUE TITLE
dieter & twitter bootstrap

### ISSUE THREAD

**Issue by DanThiffault** [edgecase/dieter#32]
I have a project working with dieter. I am currently using code from efcbb9b19b2c28c1aa0ade06ee5e6a694f6f473a.  I can create a main.less file:

``` less
@lime: #AABB00;                                                                                                                                                                                                     
body {background-color: @lime;} 
```

and using link-to-asset I can pull up /assets/stylesheets/main.css which looks like this

``` css
body {
  background-color: #aabb00;
}
```

So it looks like my overall setup is working. if I copy files from [bootstrap](https://github.com/twitter/bootstrap) in to my project and update my main.less as follows:

``` less
@lime: #AABB00;                                                                                                                                                                                                     
body {background-color: @lime;} 

@import "bootstrap.less"
```

I get the error:

```
org.mozilla.javascript.JavaScriptException: ERROR:null:null:-1: 
"error" is not defined.
TYPE: Syntax
INDEX: undefined
EXTRACT: ,, (less-wrapper.js#64)
```

Reading through the bootstrap forums it looks like people were having incompatibilities with versions <1.3 of the less compiler.  If I run lessc main.less the file properly compiles.

It looks to me like dieter uses v1.3 of the js compiler executed using less-wrapper.js.  Did I get all of this correct? I'll keep digging in to see if I can track it down,  but any suggestions/directions to look would be much appreciated.

**Comment by pbiggar** [comment]
I saw exactly the same thing. For me, it was a problem where I had a nested import, and the inner import did not use the correct path structure (which seems to have changed: it was relative to the root, its now relative to the importing file).

To help debug this, I would use wget to fetch files directly while commenting out half the file at once to find the line with the error, and then I experimented to find the right directory structure. I use the v8 branch (unstable - dont use in production!) to speed things up. Hope that helps.
